packages/neuron-wallet/src/services/cells.ts

```diff
@@ -71,23 +73,23 @@ export default class CellsService {
     for (const output of cells) {
       if (output.depositOutPoint) {
         const tx = txs.filter(t => t.hash === output.depositOutPoint!.txHash)[0]
-        if (tx) {
-          output.depositTimestamp = tx.timestamp
+        if (tx && tx.timestamp) {
+          output.setDepositTimestamp(tx.timestamp)
         }
       }
     }
 
     return cells
   }
 
-  public static getLiveCell = async (outPoint: OutPoint): Promise<Cell | undefined> => {
+  public static getLiveCell = async (outPoint: OutPoint): Promise<Output | undefined> => {
```

Comment:
As above, please change name accordingly now that it has a different return type.